genericjpa/README.md

```diff
@@ -0,0 +1,84 @@
+Hibernate-Search-JPA
+====================
```

Comment:
oh, this is a relic from the old version that had its own repository.

As of now the async backend is a side product of my genericjpa efforts and is therefore in the genericjpa folder.